src/buildercore/bootstrap.py

```diff
@@ -256,6 +272,14 @@ def update_sqs_stack(stackname, **kwargs):
         # stack has/had queues that may need to be created/destroyed
         setup_sqs(stackname, current_context_sqs, current_context['project']['aws']['region'])
 
+def update_terraform_stack(stackname, **kwargs):
+    if 'FASTLY_API_KEY' not in os.environ:
+        raise ConfigurationException("You must provide a FASTLY_API_KEY environment variable to provision Fastly resources")
+    
+    working_dir = join(TERRAFORM_DIR, stackname)
```

Comment:
every stack has its own `.cfn/terraform/api-gateway--prod/` directory for maximum isolation. May require some duplication of values and/or some basic stacks like `bus` to be created in general, but avoids a `terraform destroy` on a testing stack accidentally taking down production

Reply:
Also, everything in that directory is local. It should use a remote storage like the `elife-builder` S3 bucket to share what has been created between builder users (even Alfred)